### DISCUSSION TITLE
I wan to convert Loaded<Entity> type to Entity type

### DISCUSSION THREAD

**Discussion by jeongsik-woo** [mikro-orm/mikro-orm#5131] [Q&A] [answered]
Hi I wan to convert Loaded<Entity> type to Entity type
But I can not find any information about that.

below is example code.

```
import { MikroORM, Entity, PrimaryKey, Property, BaseEntity } from '@mikro-orm/core';

@Entity()
class YourEntity extends BaseEntity {
  @PrimaryKey()
  id!: number;

  @Property()
  someProperty!: string;
}

(async () => {
  const orm = await MikroORM.init({
    entities: [YourEntity],
    dbName: 'your_database_name',
    type: 'mysql',
  });

  const em = orm.em;

  const loadedEntity = await em.findOne(YourEntity, { id: 1 });

  // How can I get YourEntity type not Loaded<YourEntity>
})();
```

**Comment by B4nan** [comment, marked as answer]
What do you actually want to do with it? It does not make much sense to convert that to entity type (it actually is intersected with the entity type already). Maybe you want to serialize the entity to a DTO? That way you also get rid of the ORM-specific constraints like `Collection` or `Reference` abstractions.

Anyway, you can use `FromEntityType<T>` type for this, but its exactly the same as if you just do `loadedEntity as YourEntity`.
